packages/components/src/components.d.ts

```diff
@@ -217,6 +217,10 @@ export namespace Components {
         "setContents": (contents: string) => Promise<string>;
     }
     interface StencilaExecutableDocumentToolbar {
+        /**
+          * Frequency in milliseconds with which to check the status of
+         */
+        "jobPollFrequency": number;
```

Comment:
Sorry, my  previous message was not clear. What I think is needed is for the polling to continue after a connection has been made to the session, but at a possibly reduced frequency. This will allow updating of the status message e.g. "Session ended due to inactivity", "Session was stopped because it reached it's time limit". So perhaps we need `jobPreConnectPollFrequency` and `jobPostConnectPollFrequency`

Reply:
As discussed I've removed these changes and will add them in a separate PR in order to unblock the release of the changes around the inclusion of credentials with XHR requests.